pkg/virt-operator/creation/rbac/apiserver.go

```diff
@@ -148,6 +148,19 @@ func newApiServerClusterRole() *rbacv1.ClusterRole {
 					"watch",
 				},
 			},
+			{
+				APIGroups: []string{
```

Comment:
Do we miss the permissions for virt-controller? I could only find permissions for virt-api and virt-handler.

Reply:
`virt-controller` already has the proper permissions https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-operator/creation/rbac/controller.go#L135-L143

Reply:
Great. Apparently they are too wide by default. We can consider moving the operator CR into another apiresource group when we go to v1, so that it does not get implicitly the rights to modify CRs, which it does right now.